client/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+//! Exposes svc-pricing Client Functions
+
+/// Pricing Client Library: Client Functions, Structs
+pub mod flights {
```

Comment:
I think this name is not used because the library is in the manifest under a different name, but should still be something like "PricingGrpc"

Reply:
[6a33809](https://github.com/Arrow-air/svc-pricing/pull/1/commits/6a338093d439d49f74caa8e188ac2a133ea535a5)